lib/extensions/devise_async_email/devise/models/authenticatable.rb

```diff
@@ -2,7 +2,12 @@
 module Extensions::DeviseAsyncEmail::Devise::Models::Authenticatable
   module PrependMethods
     def send_devise_notification(notification, *args)
-      devise_mailer.send(notification, self, *args).deliver_later
+      deliver_queue = if [:confirmation_instructions, :reset_password_instructions].include?(notification)
```

Comment:
What other devise notifications go through this function?

Reply:
According to a quick search of the Devise repo, the following:
- email change
- password change
- unlock account instructions (I don't think we are using `lockable` feature)

Reply:
Sounds like we can just make everything inside this devise mailer to `:highest` without needing to check notification content then?